service/pom.xml

```diff
@@ -134,6 +134,12 @@
             <version>1.9.5</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.opentestsystem.common</groupId>
+            <artifactId>tds-common-legacy</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

Comment:
Also caught a duplicate declaration of `maven-compiler-plugin`; I'll clean that up too while I'm in there.